ui_testing/pages/order_page.py

```diff
@@ -208,12 +230,29 @@ def set_test_date(self, date='', row_id=None):
         test_date.send_keys(date)
         return date
 
-    def set_shipment_date(self, date=''):
-        if not date:
-            date = self.get_random_date()
-        self.base_selenium.set_text(element='order:shipment_date', value=date)
+    def get_shipment_date(self, row_id=None):
```

Comment:
why  this function is refactored in this test case ?
LIMS-3406 has no relation with shipment date

Reply:
Done